client/my-sites/menus/menu-editable-item.jsx

```diff
@@ -16,17 +16,25 @@ var siteMenus = require( 'lib/menu-data' ),
 		CategoryOptions = require( './item-options/category-options' ),
 		PostList = require( './item-options/post-list' ),
 		MenuPanelBackButton = require( './menu-panel-back-button' ),
-		analytics = require( 'analytics' );
+		analytics = require( 'analytics' ),
+    Gridicon = require( 'components/gridicon' );
 
 /**
  * Components
  */
 var Button = React.createClass({
 	render: function() {
-		var className = ( this.props.className || '' ) + ' button';
+		var className = ( this.props.className || '' ) + ' button',
+        icon =this.props.icon, gridiconButton = "";
```

Comment:
Local variables should be declared using `var` (or `let`/`const`). Check the Calypso [variables style](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#declaring-variables-with-var) guide.